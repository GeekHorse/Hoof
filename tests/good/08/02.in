new right two done
new down three done
new down four done
most up
new up one done
save
