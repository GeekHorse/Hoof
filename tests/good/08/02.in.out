new right one done
new down two done
new down three done
new down four done
